tests/suites/0_stateless/05_ddl/05_0000_ddl_create_tables_v2.sql

```diff
@@ -78,3 +79,8 @@ DROP DATABASE db1;
 DROP DATABASE db2;
 
 CREATE TABLE system.test; -- {ErrorCode 1002}
+
+-- create table with column comment
+-- SELECT '====CREATE TABLE WITH COLUMN COMMENT====';
+-- CREATE TABLE column_comment_test (a INT COMMENT 'comment for a', b FLOAT NULL DEFAULT 0 COMMENT 'comment for b');
```

Comment:
Why disable the test here? If we need `enable_planner_v2` , please add a new test file for it :)

Reply:
Or the test_stateless_cluster_* will failed.

Reply:
We should deep dive why the cluster has errors

Reply:
I don't know if it is because https://github.com/datafuselabs/databend/pull/4544 is not finished.

Reply:
Not sure why the create table need cluster, what's the error is?

Reply:
```
ERROR 1105 (HY000) at line 61: Code: 1005, displayText = error: 
  --> SQL:1:14
  |
1 | create table with column comment
  | ------       ^^^^ expected <Ident>, <QuotedString>, or `IF`
  | |             
  | while parsing `CREATE TABLE [IF NOT EXISTS] [<database>.]<table> [<source>] [<table_options>]`

.
ERROR 1105 (HY000) at line 63: Code: 1022, displayText = COMMENT 'comment for a' column option is not supported, please do not specify them in the CREATE TABLE statement.
ERROR 1105 (HY000) at line 64: Code: 1025, displayText = Unknown table 'column_comment_test'.
```

Do cluster and standalone use the different parser?

Reply:
```
ERROR 1105 (HY000) at line 62: Code: 1022, displayText = COMMENT 'comment for a' column option is not supported, please do not specify them in the CREATE TABLE statement.
ERROR 1105 (HY000) at line 63: Code: 1025, displayText = Unknown table 'column_comment_test'.
```

This will not happen in standalone test.

Do standalone and cluster use different parsers?

Reply:
If it is because the sqls will not be executed on the same node, and `set enable_planner_v2 = 1` don't take effect on every node?

And `CREATE TABLE column_comment_test (a INT COMMENT 'comment for a', b FLOAT NULL DEFAULT 0 COMMENT 'comment for b')` is parsed on a node with planner_v1?

Reply:
@RinChanNOWWW I think the reason is that we won't enable new planner in cluster mode for now, see https://github.com/datafuselabs/databend/blob/04a6c54fb958e424d2cacdb06eec09e0360525eb/query/src/servers/mysql/mysql_interactive_worker.rs#L301-L302

It is because new planner haven't implemented distributed query yet, which we are working on recently.

To resolve the test issue, you can create a particular result file for cluster mode, just as https://github.com/datafuselabs/databend/blob/main/tests/suites/0_stateless/04_explain/04_0000_explain_cluster.result